content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -205,7 +203,12 @@ it with other people.
 
 If you wish to change the user you have authenticated with, or for
 troubleshooting misc. token errors, simply remove the user credentials JSON file
-and authorize again.
+and authorize again. You might also want to use `profile` parameter to make DVC
+cache your credentials per-`profile` instead of per-`client_id`:
```

Comment:
```suggestion
and authorize again. Alternatively you can use a custom `profile`-based
credential cache (instead of the default, `gdrive_client_id`-based one):
```